packages/pigeon/example/app/macos/Runner/messages.g.m

```diff
@@ -218,13 +235,24 @@ - (instancetype)initWithBinaryMessenger:(NSObject<FlutterBinaryMessenger> *)bina
   self = [super init];
```

Comment:
While it probably doesn't matter much here, the expectation in Obj-C (which can matter in subclassing, for instance) is that this method would delegate to the more-argument-rich version of init. So `return [self initWithBinaryMessenger:binaryMessenger messageChannelSuffix:@""];`